<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFocusOnStudyClass</class>
 <widget class="QWidget" name="CFocusOnStudyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFocusOnStudy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_studyPagePushButton">
       <property name="text">
        <string>学习</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_plantPagePushButton">
       <property name="text">
        <string>花园</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_managePushButton">
       <property name="text">
        <string>我</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="m_studyStackedWidget">
     <widget class="QWidget" name="m_studyPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2">
        <widget class="QPushButton" name="m_studyPushButton">
         <property name="text">
          <string>开始学习</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2" colspan="3">
        <widget class="QLCDNumber" name="m_lcdNumber"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="m_minuteSpinBox">
         <property name="suffix">
          <string>分钟</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="m_hourSpinBox">
         <property name="suffix">
          <string>小时</string>
         </property>
         <property name="maximum">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_plantPage"/>
     <widget class="QWidget" name="m_managePage">
      <widget class="QLabel" name="m_manageUsernameLabel">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="m_modPasswordPushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_switchUserPushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>切换用户</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FocusOnStudy.qrc"/>
 </resources>
 <connections/>
</ui>
